### ISSUE TITLE
Allow DOTweenAnimation to specify starting value

### ISSUE THREAD

**Issue by NibbleByte** [Demigiant/dotween#513]
Hello,

We consider this to be a missing feature. DOTweenAnimation should allow (optionally) to specify what value it should start from instead of the current value left in the editor. Here is a simple use case:
- I have a white button in a normal state.
- It has a DOTweenAnimation that when enabled it should flash the button from dark orange to light orange.
- DOTweenAnimation is used so the artist can set whatever he likes animation without touching the code.

At the moment it seems impossible to set the starting value (no matter TO/FROM) of the tween. It will always flash from white to orange. There are ways to go around this, but it would be simpler if DOTweenAnimation supported this.

We ended up extending it, adding "Start Value" option. It hooks up right before tween creation and sets the specified value, so the DOTween created finds this as initial value. It is kind of a hack indeed. Had to fix the preview as well to restore it on stop. 
![image](https://user-images.githubusercontent.com/2332996/133921661-c5e73fce-18ff-4ecc-a8a2-11f4eb469978.png)

DOTween v1.2..632
DOTweenPro v1.0.310

Cheers

**Comment by Demigiant** [comment]
Ahoy! Adding this to my TODO list :)

**Comment by allenz8512** [comment]
![Snipaste_2022-08-28_18-03-30](https://user-images.githubusercontent.com/3109032/187066373-2bb1393c-b460-403e-b09f-90a3adc0d79d.png)

same issue，we even add feature to set value on tween complete,  for using dotween for interruptable ui animation

**Comment by seoyeon01** [comment]
Has this function been updated?
Can I specify Start Value?

**Comment by Eugen1344** [comment]
I need this feature too, was Googling to find it

**Comment by LokoJigen** [comment]
Hello, I just started coding a solution and as usual I thought to search before reinventing the wheel. Anybody pushed the wheel?

**Comment by Danielfib** [comment]
Also would love this feature!
Will likely end up having to extend the component as well, as NibbleByte did.

**Comment by hbarkine** [comment]
This would be really great and essential in my opinion. Im really trying to use Dotween Animation component in my workflow as much as possible. And this is actually lacking.
Yet thanks for the great asset since this is my first comment in this repo!

**Comment by MattLP** [comment]
Yes that'd be amazing to have that as well, I'm having issue with a monobehavior wrapper that I wrote which I allow it to kill and re-create the tween on demande, for unfrequently accessed UIs, but not having a way to Reset by code is making my life quite hard.

For example, my coworkers used a scale From 1, 0, 1, and the transform at editor time is 1, 1, 1, but when I Open it, I play Forward and when I close it, I play backwards then kill the tween, but then the Scale value remains now at 1, 0, 1 and that UI goes into the Pool, so next time it's brought up, it does nothing because it's now "initial value" when I dynamically create the tween is at 1, 0, 1 already.
